File: SEIST/polyglot/julia/readiness_metrics.jl
module SeisReadinessMetrics

const readiness_metrics = (
    reduced_motion_required = true,
    dependency_budget_locked = true,
    live_upload_requires_target = true,
)

is_ready(metrics = readiness_metrics) =
    metrics.reduced_motion_required &&
    metrics.dependency_budget_locked &&
    metrics.live_upload_requires_target

end

File: SEIST/polyglot/r/readiness_metrics.R
seis_readiness_metrics <- list(
  reduced_motion_required = TRUE,
  dependency_budget_locked = TRUE,
  live_upload_requires_target = TRUE
)

seis_ready <- function(metrics = seis_readiness_metrics) {
  isTRUE(metrics$reduced_motion_required) &&
    isTRUE(metrics$dependency_budget_locked) &&
    isTRUE(metrics$live_upload_requires_target)
}
